.github/workflows/build_and_test_dbt.yaml

```diff
@@ -61,16 +81,21 @@ jobs:
       - name: Test models
         run: |
           if [[ $MODIFIED_RESOURCES_ONLY == 'true' ]]; then
-            echo "Running tests on modified/new resources only"
-            dbt test -t "$TARGET" -s state:modified state:new --defer --state "$STATE_DIR"
+            if [[ $MANUALLY_DISPATCHED == 'true' ]]; then
+              echo "Running tests on manually selected resources"
+              dbt test -t "$TARGET" -s ${{ inputs.models }} --defer --state "$STATE_DIR"
+            else
+              echo "Running tests on modified/new resources only"
+              dbt test -t "$TARGET" -s state:modified state:new --defer --state "$STATE_DIR"
+            fi
           else
             echo "Running tests on all resources"
             dbt test -t "$TARGET"
```

Comment:
**[Question, non-blocking]** Same question here as above RE: fallback behavior!

Reply:
I think [this comment](https://github.com/ccao-data/data-architecture/pull/142#discussion_r1338794842) resolves my concern here 👍🏻